general/mem_manager.cpp

```diff
@@ -56,6 +56,7 @@ MemoryType GetMemoryType(MemoryClass mc)
       case MemoryClass::HOST_32: return MemoryType::HOST_32;
       case MemoryClass::HOST_64: return MemoryType::HOST_64;
       case MemoryClass::DEVICE:  return mm.GetDeviceMemoryType();
+      case MemoryClass::DEVICE_TEMP:  return mm.GetDeviceTempMemoryType();
```

Comment:
Is `TEMP` vs `POOL` more understandable?

Reply:
Do you mean `TEMP` or `POOL` vs `DEVICE_TEMP`? Or `DEVICE_TEMP` vs `DEVICE_POOL`?